core/domain/html_validation_service.py

```diff
@@ -116,11 +116,13 @@ def wrap_with_siblings(tag, p):
 
 # List of oppia noninteractive inline components.
 INLINE_COMPONENT_TAG_NAMES = (
-    rte_component_registry.Registry.get_inline_components())
+    rte_component_registry.Registry.get_component_tag_names(
```

Comment:
Ah, no, keep get_inline_components() as part of the external API for the registry. Have that internally call _get_component_tag_names(A, B). Ditto below for get_block_components() and get_simple_components().

Reply:
Done!